setup.cfg

```
@@ -21,7 +21,7 @@ ignore-path = _build,docs/_build
 [isort]
 line_length=100
 multi_line_output=4
-skip=migrations,project_template,node_modules,.git,__pycache__,LC_MESSAGES
+skip=migrations,project_template,node_modules,.git,__pycache__,LC_MESSAGES,venv
```

Hmm, I can see why we might want to do this but it’s unrelated to the rest of the PR so would be better separately.

```suggestion
skip=migrations,project_template,node_modules,.git,__pycache__,LC_MESSAGES
```

---

I have reverted that in 4247756